include/miopen/miopen.h

```diff
@@ -346,6 +346,11 @@ MIOPEN_DECLARE_OBJECT(miopenDropoutDescriptor);
  */
 MIOPEN_DECLARE_OBJECT(miopenReduceTensorDescriptor);
 
+/*! @ingroup mha
+ * @brief Creates the miopenMHADescriptor_t type
+ */
+MIOPEN_DECLARE_OBJECT(miopenMHADescriptor);
```

Comment:
[Recommendation] Here and everywhere: Despite the fact that the old APIs are done differently, I prefer keeping camelCase even for acronyms:
```suggestion
MIOPEN_DECLARE_OBJECT(miopenMhaDescriptor);
```
What do you think?

Reply:
[resolved] @Vsevolod1983 pls. hide this comment